### ISSUE TITLE
lots of `wmllint` warnings

### ISSUE THREAD

**Issue by cooljeanius** [knyghtmare/The_Eight_of_Cembulad#7]
Should I paste them all here, or just leave a summary?

**Comment by knyghtmare** [comment]
I think I am aware of them.
mostly FOREACH and NEXT

**Comment by cooljeanius** [comment]
the ones I'm getting are mostly `unit speaks in his/her "die" event rather than "last breath"` and `unknown '%s' referred to by id`

**Comment by knyghtmare** [comment]
> the ones I'm getting are mostly `unit speaks in his/her "die" event rather than "last breath"` and `unknown '%s' referred to by id`

Hmm...I think this is related to the convention followed in 1.8/1.10...nowadays we use last breath for messages and die event to throw a endlevel defeat. Needs updating I guess

**Comment by cooljeanius** [comment]
yeah I had to do a lot of fixing those for The War of Terrador, too
